proxy/api/src/project.rs

```diff
@@ -53,7 +53,26 @@ pub struct Project {
     /// Attached metadata, mostly for human pleasure.
     pub metadata: Metadata,
     /// High-level statistics about the project
-    pub stats: coco::Stats,
+    pub stats: Option<coco::Stats>,
```

Comment:
I'm not _too_ happy with this. I might split things out into `Partial` and `Project` where a `Partial` transitions to `Project` when it can fulfill the data requirements, i.e. `coco::Stats` in this case.

Reply:
I think it's ok for now, until we have other states. But the stats are not crucial to the project, so it makes sense.